synapse/config/server.py

```diff
@@ -351,6 +375,20 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #  - nyc.example.com
         #  - syd.example.com
 
+        # Prevent federation requests from being sent to the following
+        # blacklist IP address CIDR ranges.
+        #
+        #federation_ip_range_blacklist:
```

Comment:
do you think we should uncomment this so that we blacklist the RFC1918 addresses by default for new deployments?

Reply:
Seeing as it would be a security benefit for the vast majority of use cases, I would argue yes.

Reply:
so would I. Please can you make it so?

Reply:
Done so, but requires having sytest override it so that tests don't fail.